### ISSUE TITLE
Curriculum Design: Crypto-economics & Token Engineering

### ISSUE THREAD

**Issue by trainfarb** [TheSchoolofBitcoin/curriculum#2]
### Goals
1. Introduce financial literacy course as introduction to TSOBTC’s DAES
2. As we create the course, also define the DAO tokenomics so we can attract more contributors
3. Stacks community engagement through the minting of free (clarity) NFT course certificates

### Approach
- Define mission/vision of course
- Outline community project for the course (ex. learners contributing to DAES tokenomics)
- Develop course specific brand guidelines
- Create marketing strategy for the course
- Develop NFT minting dApp for course completion
- Write Litepaper that outlines the goals of the course
- Write storyboard for mythology/comic based on the Litepaper
- Create a comprehensive course that learners will follow
- Develop a game to engage learners throughout the course
- Keep track of all notes so others can follow in our footsteps

### Resources
- [Token Engineering Community](https://tokenengineeringcommunity.github.io/website/docs/getting-started-welcome):
- [Platform Design Toolkit](https://boundaryless.io/pdt-toolkit/)
- Etc.

**Comment by electrafrost** [comment]
Excellent outline to build on :) What is the age group for participation? ...or could there be different entry levels based on age?

**Comment by kdnolan** [comment]
good point.. we could do basic language for younger learners maybe? with
the same outcome?

**Comment by trainfarb** [comment]
@electra-frost @kdnolan yea that sounds good! Maybe we can find a way to encourage the community to create courses for different audiences (I complete the course and then find ways to explain it to others in an easier way to solidify my learning). I really like this WIRED series on explaining things at 5 diff levels of difficulty. Here’s on explaining blockchain in 5 diff ways: https://youtu.be/hYip_Vuv8J0

**Comment by kdnolan** [comment]
That is a great video!

I like that idea - learners teaching learners was def the end goal, we may
as well jump off with that hey? :)
